packages/cli/lib/utils.js

```diff
@@ -103,6 +104,32 @@ exports.validateClassName = function(name) {
   return util.format('Class name is invalid: %s', name);
 };
 
+exports.remindAboutNamingRules = function(name, log) {
+  if (name.includes('_')) {
+    this.log(
+      chalk.red('>>> ') +
+        `Underscores _ in the class name will get removed: ${name}`,
+    );
+  }
+  if (name.match(/[\u00C0-\u024F\u1E00-\u1EFF]/)) {
+    this.log(
+      chalk.red('>>> ') +
+        `Accented chars in the class name will get replaced: ${name}`,
+    );
+  }
+};
+
+exports.printClassFileName = function(type, typePlural, name, log) {
```

Comment:
printClassFileName => logClassGeneration?

Reply:
I'll go with `logClassCreation` to match the message printed. I find `generation` ambiguous - is it describing the process of generating something new, or a generation like a group of similarly-aged organisms?